chat-langchain/app/components/ChatMessageBubble.tsx

```diff
@@ -347,21 +347,6 @@ export function ChatMessageBubble(props: {
             >
               👎
             </Button>
-            <Spacer />
```

Comment:
Please bring back the closing tag - Spaser

Wouldn't it be better to comment the code out, so that we could bring it back if needed?

Reply:
From Katerina on slack:
```
One small comment on the PR

Oops, I might have misread the <Spacer />

Then, disregard
```